common/src/main/java/whocraft/tardis_refined/common/network/messages/ChangeDesktopMessage.java

```diff
@@ -52,9 +53,11 @@ public void handle(MessageContext context) {
         level.ifPresent(x -> {
             TardisLevelOperator.get(x).ifPresent(operator -> {
                 TardisPilotingManager pilotManager = operator.getPilotingManager();
+                TardisInteriorManager interiorManager = operator.getInteriorManager();
 
-                if (!pilotManager.isInFlight()) {
-                    operator.getInteriorManager().prepareDesktop(desktopTheme);
+                if (!pilotManager.isInFlight() && interiorManager.hasEnoughFuel()) {
```

Comment:
We should maybe play a sound when the desktop cannot be changed?

Reply:
Good point i'll add some temporary nuh uh sound